redux_userlist_06/src/components/Dashboard/userProfile.tsx

```diff
@@ -1,21 +1,37 @@
+import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
 import { UserDetails } from "../../interfaces";
 
 import "./styles.css";
 
-const UserProfile = (props: any)=> {
+const UserProfile = (props: any) => {
   const details: UserDetails = props.data;
+  const [visible,setVisible] = useState(false);
+  const handleTouch = () => {
+    setVisible(true)
+  };
 
+  const handleTouchCancel = ()=>{
+    setVisible(false)
+  }
   return (
-    <Link to="/updateUser" state={{data:props.data}}>
-      <div className="profile">
+    <Link to="/updateUser" state={{ data: props.data }}>
+      <div
+        className="profile"
+        onMouseOver={()=>handleTouch()}
```

Comment:
why extra callback function
onMouseOver={handleTouch}
this will also work!....